eng/restore-toolset.ps1

```diff
@@ -26,8 +26,7 @@ function InitializeCustomSDKToolset {
   InstallDotNetSharedFramework "5.0.0"
   InstallDotNetSharedFramework "6.0.0"
   InstallDotNetSharedFramework "7.0.0"
-  InstallDotNetSharedFramework "8.0.0-preview.7.23375.6"
-  InstallDotNetSharedFramework "8.0.0-rc.2.23457.7"
+  InstallDotNetSharedFramework "8.0.0-rc.2.23479.6"
```

Comment:
Do we bring the 8.0.0 shared framework in for testing?

Reply:
Correct, we install each of the downlevel runtimes as that includes apphosts and lets us run apps in testing. We may not need them all at this point but we'd definitely need the 8.0 one for some set of tests.

Reply:
Cool. It would be good to follow-up and discuss the ones that are already out-of-support: anything lower than 6.0.0. I think it sends a weird signal to our customers to still test against out-of-support runtimes.